README.md

```diff
@@ -1,6 +1,42 @@
 # Approved Premises API
 
-This is the backend for the Approved Premises service. Its API is consumed by the corresponding "UI" codebase ([approved-premises-ui](https://github.com/ministryofjustice/approved-premises-ui)). 
+This is the backend for the Approved Premises service. Its API is consumed by the corresponding "UI" codebase ([approved-premises-ui](https://github.com/ministryofjustice/approved-premises-ui)).
+
+## Prerequisites
+
+TBC
+
+## Setup
+
+When running the application for the first time, run the following command:
+
+```bash
+script/setup
+```
+
+If you're coming back to the application after a certain amount of time, you can run:
+
+```bash
+script/bootstrap
+```
```

Comment:
As these don't do anything yet would it be helpful to say as much, e.g. with a "TODO"